WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -367,7 +367,7 @@ public static string ToArgumentString(Network network, IEnumerable<HwiOption> op
 				{
 					argumentBuilder.Append(' ');
 				}
-				argumentBuilder.Append(command.ToString().ToLowerInvariant());
+				argumentBuilder.Append(command?.ToString().ToLowerInvariant());
```

Comment:
We are checking `if(command is { })` just 6 lines above, but IntelliSense doesn't care about it, unfortunately.

Reply:
Would `if (command is not null)` help? 

Or in `if (command is {} nonNullCommand)` the `nonNullCommand` should not be `null` :)

Reply:
That was my first try and sadly it didn't help.
But your second suggestion works, thanks for the idea!

Reply:
Well, unfortunately it's very ugly with that `nonNullCommand` but it's still probably better than `command?`.